### ISSUE TITLE
Copy / Paste from slack

### ISSUE THREAD

**Issue by ymeiner** [ndri/emojicipher#2]
With slack, you can paste emojis in but selecting and copying existing emojis gives you a text version that looks like that 

`:{something}:`

example:
`:kissing_smiling_eyes: :hot_pepper: :raising_hand::skin-tone-2: :pencil2: :cheese_wedge:`

Please allow the ability to paste copied emojis from slack.

Thank you!

**Comment by ndri** [comment]
I made it so :shortcodes: will be replaced with emojis, but there is no official list of them and the best one I managed to find was from http://emoji.codes/family. This however, did not have some of the Slack ones you posted. I added them in manually, but there are probably lots more that I don't know about. 

Do you know where Slack gets its emoji shortcodes from? Their [Help Center](https://get.slack.help/hc/en-us/articles/202931348-Emoji-and-emoticons) says [Emoji Cheat Sheet](http://www.webpagefx.com/tools/emoji-cheat-sheet/), but that seems a little dated.

**Comment by ymeiner** [comment]
Yeah, the emoji Cheat Sheet is what i know about. this will be enough for me :)
